tests/test_schema.py

```diff
@@ -138,6 +139,24 @@ def test_parallax_schema(values, error_state):
     """
     schema_tester(Parallax, values, error_state)
 
+@pytest.mark.parametrize("values, error_state",
+                         [
+                             ({"radial_velocity_km_s": 30}, None),
+                             ({"radial_velocity_km_s": -30}, None),
+                             ({"radial_velocity_km_s": None}, ValueError),
+                             ({"radial_velocity_error_km_s": None}, None),
+                             ({"radial_velocity_error_km_s": 30}, None),
+                             ({"radial_velocity_error_km_s": -30}, None),
```

Comment:
Should the uncertainty be allowed to be negative?

Reply:
Yeah I guess not! I was thinking about the "negative parallaxes are not unexpected" Gaia results, and I must've mixed them up with errors. I can fix this for parallaxes and RVs.